astroquery/dastcom5/core.py

```diff
@@ -61,21 +59,11 @@ def download_dastcom5(self):
                 os.makedirs(self.local_path)
 
             print("Downloading datscom5.zip")
-            with self._show_download_progress(unit='B', unit_scale=True, miniters=1, desc="dastcom5.zip") as t:
-                urllib.request.urlretrieve(
-                    self.ftp_url + "dastcom5.zip", filename=dastcom5_zip_path, reporthook=t.update_to)
+            self._download_file(url=ftp_path, local_filepath=dastcom5_zip_path)
```

Comment:
Looks good.  You could pass on some of the other optional keyword arguments here like `timeout`, `cache`, and `continuation` if you'd like.  But this is fine.

Reply:
I hoped it'd work fine, but it isn't on my system. I am unable to understand whether it is my internet connection and system or there is something wrong in the code